trantor/net/inner/TcpConnectionImpl.cc

```diff
@@ -197,7 +197,7 @@ void TcpConnectionImpl::extendLife()
         {
             auto timingWheelPtr = timingWheelWeakPtr_.lock();
             if (timingWheelPtr)
-                timingWheelPtr->insertEntry(idleTimeout_, entry);
+                timingWheelPtr->insertEntryInloop(idleTimeout_, entry);
```

Comment:
The enableKickingOff method is exposed to the user and we should not assume that the timing wheel is in the same event loop.